nixos/modules/services/desktops/pipewire/easyeffects.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.pipewire.easyeffects;
+in
+{
+  meta.maintainers = with lib.maintainers; [ starcraft66 ];
+
+  options = {
+    services.pipewire.easyeffects = {
+      enable = lib.mkEnableOption "Audio effects for PipeWire applications.";
+
+      package = lib.mkOption {
```

Comment:
This is a leaf package so you can easily just use overlay to change the derivation.

Reply:
I can remove this option, I just left it there because I referred to other pipewire modules to write this one.